docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md

```diff
@@ -1 +1,326 @@
+    <cpix:ContentKeyList>
+        <cpix:ContentKey kid="3e4a4bfb-a252-a4b4-c839-6c8954d986f5" explicitIV="MjlkMzNlYzVjYWIyZmRmZg==" commonEncryptionScheme="cenc">
+            <cpix:Data>
+                <pskc:Secret>
+                    <pskc:PlainValue>3wFFjPspsydY/t2uSPeE6w==</pskc:PlainValue>
+                </pskc:Secret>
+            </cpix:Data>
+        </cpix:ContentKey>
+    </cpix:ContentKeyList>
+    <cpix:DRMSystemList>
+        <cpix:DRMSystem kid="3e4a4bfb-a252-a4b4-c839-6c8954d986f5" systemId="9a04f079-9840-4286-ab92-e65be0885f95">
+            <cpix:PSSH><PlayReady_PSSH></cpix:PSSH>
+        </cpix:DRMSystem>
+        <cpix:DRMSystem kid="3e4a4bfb-a252-a4b4-c839-6c8954d986f5" systemId="edef8ba9-79d6-4ace-a3c8-27dcd51d21ed">
+            <cpix:PSSH><Widevine_PSSH></cpix:PSSH>
+        </cpix:DRMSystem>
+        <cpix:DRMSystem kid="3e4a4bfb-a252-a4b4-c839-6c8954d986f5" systemId="94ce86fb-07ff-4f43-adb8-93d2fa968ca2">
+            <cpix:URIExtXKey><FairPlay_URIExtXKey</cpix:URIExtXKey>
+            <cpix:HLSSignalingData playlist="master"><master_playlist_value></cpix:HLSSignalingData>
+            <cpix:HLSSignalingData playlist="media"><media_playlist_value></cpix:HLSSignalingData>
+        </cpix:DRMSystem>
+    </cpix:DRMSystemList>
+    <cpix:ContentKeyUsageRuleList></cpix:ContentKeyUsageRuleList>
+</cpix:CPIX>
+```
+
+#### CPIX keys as JSON
+
+Below is an example of a CPIX keys as JSON document with all `DRM Providers` selected. 
+
+```
+{
+    "key_id_hex": <key_id_hex_value>,
+    "content_key_hex": <content_key_hex_value>,
+    "iv_hex": <iv_hex_value>,
+    "playready_pssh_data": <playready_pssh_data_value>,
+    "playready_system_id": "9a04f079-9840-4286-ab92-e65be0885f95",
+    "widevine_drm_specific_data": <widevine_drm_specific_data_value>,
+    "widevine_drm_specific_data_with_key_id": <widevine_drm_specific_data_with_key_id_value>,
+    "widevine_system_id": "edef8ba9-79d6-4ace-a3c8-27dcd51d21ed",
+    "fairplay_system_id": "94ce86fb-07ff-4f43-adb8-93d2fa968ca2",
+    "fairplay_laurl": "skd://fairplay-license.vudrm.tech/v2/license/<content_id>"
+}
+```
+
+## Packaging Content with Widevine and Playready DRM
+
+Now we have retrieved the relevant documentation, we can now package content.
+
+Below is an example of how we can send a request to Shaka Packager to encrypt our content with Widevine and Playready DRM and package it:
+
+```
+  packager \
+  in=media/audio.mp4,stream=audio,output=media/encrypted-test-audio.mp4,drm_label=AUDIO \
+  in=media/source.mp4,stream=video,output=media/encrypted-test-video.mp4,drm_label=VIDEO \
+  --enable_raw_key_encryption \
+  --keys label=AUDIO:key_id=<key_id_hex_value>:key=<content_key_hex_value>,label=VIDEO:key_id=<key_id_hex_value>:key=<content_key_hex_value> \
+  --pssh <Widevine_PSSH(converted to Hex format)> \
+  --protection_systems Widevine,PlayReady \
+  --mpd_output media/encrypted-test.mpd
+```
+
+### Line 1
+
+`packager` is used to start our request.
```

Comment:
I respectfully disagree. If we are giving this for customers to use, I think it does not hurt to go as in-depth as possible. More information, the better. I feel that this leaves no questions to be asked by clients when packaging content with Shaka Packager and should make the process seamless.

However, if everyone else agrees with Perry then I can amend this.

@somerobmitchell @fschurer @rogerpales what are your thoughts?

Reply:
I'll defer to Roger and Rob on this one, but would assume that giving customers more info is a good thing (provided it doesn't mean we have to regularly check for updates to the Shaka docs in case our reference here).